ui/apps/platform/src/services/NetworkService.js

```diff
@@ -236,7 +242,10 @@ export function fetchNetworkFlowGraph(
         urlParams.includePorts = true;
     }
     // for openshift filtering toggle
-    if (localStorage.getItem(ORCHESTRATOR_COMPONENTS_KEY) !== 'true') {
+    if (
+        !includeOrchestratorComponents &&
+        localStorage.getItem(ORCHESTRATOR_COMPONENTS_KEY) !== 'true'
```

Comment:
Ohh I wasn't aware we were keeping that value in local storage. Whats the story about that?

Reply:
Localstorage for the "show orchestrator components" setting was part of the change we implemented when we joined Red Hat, so that people wouldn't see all the OpenShift namespaces in the Network Graph and listed on the Risk page. The setting is supposed to be the same on both pages, and since we were pulling back from Redux, localstorage was a good alternative.